kie-dmn-feel/src/main/java/org/kie/dmn/feel/lang/Type.java

```diff
@@ -24,4 +42,36 @@
 
     String toString( Object value );
 
+    public static Type determineTypeFromClass( Class<?> clazz ) {
```

Comment:
during FEEL-compilation-stage I inspect reflectively the fields of the JavaBacked/MapBacked return/values and I use this method to determine the FEEL-type.